scripts/traverse.js

```diff
@@ -89,21 +89,7 @@ function traverseFeatures(obj, browsers, values, depth, identifier) {
   return features.filter((item, pos) => features.indexOf(item) == pos);
 }
 
-const main = (
-  folders = [
-    'api',
-    'css',
-    'html',
-    'http',
-    'svg',
-    'javascript',
-    'mathml',
-    'webdriver',
-  ],
-  browsers = Object.keys(bcd.browsers),
-  values = ['null', 'true'],
-  depth = 100,
-) => {
```

Comment:
I don't think we should remove the default values here.  While they're mostly duplicated with `yargs`, it would be nice to provide defaults on the function call should we desire to run this function from another script.

Reply:
I was bugged that the defaults don't exactly match, but I've now backed out this change. I might be bugged again in the future and fixed it in some other way.